Modules/_bisectmodule.c

```diff
@@ -8,7 +8,7 @@ Converted to C by Dmitry Vasiliev (dima at hlabs.spb.ru).
 
 _Py_IDENTIFIER(insert);
 
-static Py_ssize_t
+static inline Py_ssize_t
```

Comment:
Perhaps you meant `Py_LOCAL_INLINE(Py_ssize_t)`?

Reply:
Is that being used anymore?   I believe we're using "inline" directly (and some other C99 features) nowadays.

Reply:
I don't know. Maybe `Py_LOCAL_INLINE` is outdated.

It is expanded to a different form on MSVC.